### PR TITLE
Added media queries to the desktop design

### PR DESCRIPTION
**Pull request by gracekabaghe**
- Added media queries for responsive design

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -11,7 +11,6 @@ body {
 }
 
 .container {
-  width: 100%;
   margin: auto;
   display: grid;
 }
```
Comment (dasileker) [inline styles.css:16]: - [ ] The width of the page is big, please resize it to fit the page.

md

File: styles.css [hunk 2]
```diff
@@ -366,3 +359,161 @@ textarea {
...
+
+@media all and (min-width: 1000px) {
+  .dsk-top-nav {
+    display: flex;
+    list-style: none;
+    gap: 20px;
+    font-family: Poppins, sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
```
Comment (dasileker) [inline styles.css:373]: - [ ] The mobile view is not correctly set, you should also add the @media query for the mobile view, and right now the view is pretty bad.

mb


mgmm

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @gracekabaghe,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (gracekabaghe) [reply styles.css:16]: Adjusted Width in the container

File: styles.css (hunk 2 above)
Comment (gracekabaghe) [reply styles.css:373]: The navigation is now responsive

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Grace Kabaghe, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**